doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+  * :ref:`nrf53_audio_app_requirements` section in the application documentation by moving the information about the nRF5340 Audio DK to `Nordic Semiconductor Infocenter`_, under `nRF5340 Audio DK Hardware`_.
+
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Added a Kconfig option :kconfig:option:`CONFIG_APP_SENSOR_SLEEP_TO` to set the sensor idling timeout before suspending the sensor.
+
+* Removed the usage of ``ml_runner_signin_event`` from the application.
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * An application log indicating that the value of a configuration option has been updated in the :ref:`nrf_desktop_motion`.
+  * Application-specific Kconfig options :ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>` to simplify the debug configurations for the Logging and Shell subsystems.
+    See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+  * Application-specific Kconfig options that define common HID device identification values (product name, manufacturer name, Vendor ID, and Product ID).
+    The identification values are used both by USB and the Bluetooth LE GATT Device Information Service.
+    See the :ref:`nrf_desktop_hid_device_identifiers` documentation for details.
+  * The :ref:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ID_INFO <config_desktop_app_options>` Kconfig option.
+    It can be used to indicate the dongle peer identity with a dedicated event.
+  * Synchronization between the Resolvable Private Address (RPA) rotation and the advertising data update in the Fast Pair configurations using the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` Kconfig option.
+  * Application-specific Kconfig options that can be used to enable the :ref:`lib_caf` modules and to automatically tailor the default configuration to the nRF Desktop use case.
+    Each used Common Application Framework module is handled by a corresponding application-specific option with a modified prefix.
+    For example, :ref:`CONFIG_DESKTOP_SETTINGS_LOADER <config_desktop_app_options>` is used to automatically enable the :kconfig:option:`CONFIG_CAF_SETTINGS_LOADER` Kconfig option and to align the default configuration.
+  * Prompts to Kconfig options that enable :ref:`nrf_desktop_hids`, :ref:`nrf_desktop_bas`, and :ref:`nrf_desktop_dev_descr`.
+    An application-specific option (:ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>`) implies the Kconfig options that enable the mentioned modules together with other features that are needed for the Bluetooth HID peripheral role.
+    The option is enabled by default if the nRF Desktop Bluetooth support (:ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>`) is enabled.
+
+* Updated:
+
+  * Implemented the following adjustments to avoid flooding logs:
```

Comment:
```suggestion
* Updated:

  * Logging mechanism by implementing the following adjustments to avoid flooding logs:
```
Style inconsistency: "Updated Improved".